src/schoology-extractor/tests/mapping/test_discussion_replies.py

```diff
@@ -13,64 +13,73 @@
 FAKE_DISCUSSION_ID = 12
 
 
-class Test_mapping_schoology_users_to_udm:
-    @classmethod
-    def setup_class(cls):
+def describe_mapping_schoology_users_to_udm():
+    def describe_given_input_is_empty():
+        def it_should_return_the_empty_DataFrame():
+            input = pd.DataFrame()
```

Comment:
@lalrg please note the change from using a test class to using "describe" and a pytest fixture.